### DISCUSSION TITLE
Creating User Accounts with Restricted Access to Specific Movies

### DISCUSSION THREAD

**Discussion by BoBBer446** [jellyfin/jellyfin#10821] [Q&A]
Hello Jellyfin Community,

I'm seeking guidance on how to create a user account in Jellyfin and restrict its access to specific movies only. For instance, I want to set up an account for my child but limit their access to the entire library, allowing them to view only certain movies. Could someone please provide step-by-step instructions or point me to relevant documentation on how to achieve this?

Thank you in advance for your assistance!

**Comment by Darkhost1999** [comment]
![image](https://github.com/jellyfin/jellyfin/assets/60384196/982573c9-8b82-4f20-a2cb-469ff6991da1)
Hamburger menu -> Dashboard -> users -> + symbol -> Parental controls
When editing Metadata for every single item individually you want your kid to have access to. Set custom rating then LOCK changes to the Metadata. This tickbox to lock changes is near the bottom of metadata

**Reply by agarlow23** [reply]
The above method works for somewhat for kids but is useless for access control in other contexts. If you have an elderly parent for example and you don't want them to sift through heaps of irrelevant content for them, it would be nice to create a tag for that person and mark their content with that tag, just like Plex does it. Currently, the above method shows anything marked "approved" but also all of those other ratings. I can't allow the "APPROVED" tag without all the rest of that content. The Jellyfin model of default allow and selectively block individual shows is the wrong approach and people have said so for years.

**Reply by Darkhost1999** [reply]
Yea, I get what you mean. I want my adult account to filter out some content. Like I don't need the kid shows in my account.
Or some movies/shows aren't for me but are for my spouse and vice versa.

**Comment by gnattu** [comment]
This should already be possible with Jellyfin 10.9:

<img width="455" alt="image" src="https://github.com/jellyfin/jellyfin/assets/12995396/b4892a58-7955-4d8b-9b22-97ca7368ec23">

If you want to test it you call try the current unstable
